src/Generation/Generator/Renderer/Public/ReturnType/From/VoidReturnType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Generator.Model;
+
+namespace Generator.Renderer.Public;
+
+internal class VoidReturnType : RenderableScalarReturnType<GirModel.Void>
```

Comment:
- Please move this class into the namespace `Generator.Renderer.Public.ReturnType`
- Please remove the class suffix `ReturnType` so the name changes e.g. from `BitfieldReturnType` to `Bitfield`